blackbox/docs/configuration.txt

```diff
@@ -1851,6 +1851,26 @@ to your config. This will prevent allocating primary and replica
 of the same shard on the same machine even if more than one instances
 running on it.
 
+Other System Configuration
+==========================
+
+Virtual Memory
+--------------
+
+CrateDB uses a hybrid ``MMap``/``NIO`` file system storage directory to store
+its indices. Out-of-memory exceptions and failing bootstrap checks may be
+caused by the operating system's default limits on ``mmap counts``
```

Comment:
not sure "mmap counts" should be a literal here. could we replace this with a description, like "default limits on `max_map_count` (i.e. the maximum number of memory map areas a process may have) being too low""